src/EditorFeatures/Test/Utilities/PatternMatcherTests.cs

```diff
@@ -528,9 +512,10 @@ private static IEnumerable<PatternMatch> TryMatchMultiWordPattern(string candida
         {
             MarkupTestFile.GetSpans(candidate, out candidate, out ImmutableArray<TextSpan> expectedSpans);
 
-            var matches = new PatternMatcher(pattern).GetMatches(candidate, includeMatchSpans: true);
+            var matches = ArrayBuilder<PatternMatch>.GetInstance();
```

Comment:
❔ Is the general practice in tests to ignore calling `Free` on these and/or it's a "either way; doesn't matter"?